libkbfs/keybase_daemon_local.go

```diff
@@ -535,7 +535,7 @@ func (k *KeybaseDaemonLocal) addTeamKeyForTest(
 func (k *KeybaseDaemonLocal) addTeamsForTestLocked(teams []TeamInfo) {
 	for _, t := range teams {
 		k.localTeams[t.TID] = t
-		k.asserts[string(t.Name)] = t.TID.AsUserOrTeam()
+		k.asserts[string(t.Name)+"@team"] = t.TID.AsUserOrTeam()
```

Comment:
Does this go to the same place as `t_alice@twitter` or is it a completely different thing?